docs/rest.md

```diff
@@ -540,3 +540,33 @@ Get an icon from a snap installed on the system. The response will be the raw
 contents of the icon file; the content-type will be set accordingly.
 
 This fetches the icon from the package itself.
+
+## /1.0/capabilities
+### GET
+
+* Description: List of snappy capabilities
+* Authorization: guest
+* Operation: sync
+* Return: information about all of the snappy capabilities 
```

Comment:
Could you expand on this? Are we going to return "capabilities", without mentioning snappy or are we going to return a subset of them? Or perhaps are you referring to the list vs object?